### ISSUE TITLE
meshio-convert: cell data not transferred from exo or vtu to abaqus inp

### ISSUE THREAD

**Issue by gahansen** [nschloe/meshio#432]
This issue may be related or similar to closed issue #401 . When processing either exodus or vtu files into Abaqus inp files (meshio-convert fblock_restart.exo --output-format abaqus fblock_restart.inp  -or- meshio-convert fblock_restart.vtu --output-format abaqus fblock_restart.inp), none of the cell data present is read from the original file and transferred to the inp file. This can be duplicated with the fblock_restart.exo file attached to #401 .

**Comment by nschloe** [comment]
That's right. I have no idea how cell data works in Abaqus inp files. If you find a format specification, let me know.

**Comment by gahansen** [comment]
Hi Nico! I have attached an example file that I generated that might be helpful. The sections of interest are at the bottom. 

The elements are numbered from 1 to the total number of elements in the ELEMENT list. At the bottom of the file, there are element set (ELSET) commands that name each subset of elements (eg., ES_Volume 1_MAT0), and that give the range of the elements that belong in the subset (eg., 1,1481,1  The "1" at the end is the stride I believe. Immediately following that line, there is a SOLID SECTION command that associated the element set name with a material. This continues for all the subsets until all the elements have been binned into ELSETS and associated with materials.

After this, there are MATERIAL lines that associate the material name with a DENSITY, and perhaps an ELASTIC model that describes the material behaviour. I suspect things like TEMPERATURE are also possible but not in this example. We probably do not need to handle everything that could arise for my use case, as I push this forward if I encounter a situation that does not work I would just post another issue with a description of the case.

[abaqus.inp.zip](https://github.com/nschloe/meshio/files/3483540/abaqus.inp.zip)
